chain-signatures/node/src/mesh/connection.rs

```diff
@@ -35,6 +35,7 @@ impl Pool {
             }
         }
 
+        tracing::debug!("pinging participants");
```

Comment:
these stop debug logs should not be necessary. Pinging connections can never fail so we'll always call into the following debug log